server/src/main/java/info/doseamigos/doseevents/DoseEvent.java

```diff
@@ -0,0 +1,90 @@
+package info.doseamigos.doseevents;
+
+import info.doseamigos.amigousers.AmigoUser;
+import info.doseamigos.meds.Med;
+
+import java.util.Date;
+import java.util.List;
+import java.util.Objects;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * POJO for holding events that took place.  For example, if "Bobby" took "Allegra" today at 3PM, this would
+ * hold that AmigoUser, a date representing today at 3PM, the list of medicines that user took (just Allegra in this
+ * case), and an event type of "Took Medicine".
+ */
+public class DoseEvent {
+
+    private final long eventId;
```

Comment:
When I wrote the objects, I wrote them to be simple POJOs since I don't know which jackson modules the Amazon Lambda stuff includes (outside my jurisdiction AFAIK).  I don't want to use Optional on fields, as that's not how Java 8 want's them used (makes them non-serializable).  For now, I figure simple mutable objects for things we expect from PUTs and POSTs, which is everything but dose events.  I made those immutable since they're unchanging in nature.